packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -160,5 +160,11 @@ document.addEventListener('keydown', e => {
         window.__actionsForMenu.undo();
       }
     }
+  } else if (e.key === '?') {
+    e.preventDefault();
+    const isAccounts = window.location.pathname.startsWith('/accounts');
```

Comment:
We could probably do this check inside the KeyboardShortcuts component. You can use react router to get the current path and return different component based on the path.